deployments/Dockerfiles/localnode/app.py

```diff
@@ -218,7 +218,10 @@ def hard_reset() -> Tuple[Any, int]:
             if IS_DEV_MODE:
                 period_dumper.dump_period()
 
-            tendermint_node.prune_blocks()
+            return_code = tendermint_node.prune_blocks()
+            if return_code:
+                tendermint_node.start(start_monitoring=perform_monitoring)
+                raise RuntimeError("Could not perform `unsafe-reset-all` successfully!")
```

Comment:
why start the Tendermint node right before raising a `RuntimeError`?

Reply:
The fact that we did not manage to perform the `unsafe-reset-all` does not mean that we should let the node stopped.